docs/content/cluster-scope/add_operator.md

```diff
@@ -0,0 +1,32 @@
+# Constructing a PR to add an Operator
+
+There are two methods one can follow to add an Operator to a cluster:
+1. Install via OLM
+1. Manual install
+
+## Install an operator via OLM
+
+Operator Lifecycle Manager (OLM) makes it easy to install and manage Operators on OpenShift clusters.
+To install an Operator via OLM, you must make a PR that does the following:
+
+* Create the `Subscription` to the Operator
```

Comment:
I think we should give more info here, or link to where they can find more information (e.g. [here](https://olm.operatorframework.io/docs/concepts/crds/subscription/)). 

Also state that we should keep `installPlanApproval: Automatic`
And always choose the most stable option unless there's a specific reason not to 
Avoid adding a starting CSV, since we keep automating install plans, it doesn't make sense to pin it to a starting csv